exchange/bidder_validate_bids.go

```diff
@@ -8,9 +8,9 @@ import (
 
 	"github.com/mxmCherry/openrtb"
 	"github.com/prebid/prebid-server/adapters"
-	"github.com/prebid/prebid-server/currencies"
+	"github.com/prebid/prebid-server/currency"
 	"github.com/prebid/prebid-server/openrtb_ext"
-	"golang.org/x/text/currency"
+	go_currency "golang.org/x/text/currency"
```

Comment:
Underscores are not recommended for names per the Go standard. How about `goCurrency`, `golangCurrency`, `stdCurrency`, ...